optimum/gptq/quantizer.py

```diff
@@ -664,8 +684,8 @@ def load_quantized_model(
             picked contains `"disk"` values.
         disable_exllama (`Optional[bool]`, defaults to `None`):
             Whether to use exllama backend. Only works with `bits` = 4.
-        disable_exllama (`bool`, defaults to `False`):
-            Whether to use exllamav2 backend. Only works with `bits` = 4.
+        exllama_config (`Dict[str, Any]`, *optional*):
+            The exllama config. You can specify the version of the exllama kernel through the `version` key. Defaults to `{"version": 2}` if unset.
```

Comment:
How do we select cuda/cuda-old kernel?

Reply:
You set `disable_exllama` = True.

Reply:
Oh I see, so the `disable_exllama` is disable exllama altogether (v1/v2?)?

Reply:
Yes !

Reply:
Makes sense!